README.md

```diff
@@ -1 +1,25 @@
-# image-lambda
\ No newline at end of file
+# Project: Image Lambda
+
+- [GitHub Repository]()
+
+## How to use
+
+- Upload an image to the bucket and resized image will be in bucket
+
+## Lambda Function Source Code
+
+- The lambda function code provided by Amazon [tutorial](https://docs.aws.amazon.com/lambda/latest/dg/with-s3-example.html)
+
+## Lambda Use
+
+- Upload an image to [S3 Bucket]()
+- Access thumbnail image in [S3 Bucket]()
+
+## Issues During Deployment
+
+Still working on it
```

Comment:
When you have a chance to come back to this, and work on getting the image uploaded I will regrade for you. Even if the images aren't added I will give you points if you add more into issues that you are having with deployment.